extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -19,16 +19,35 @@
 
 if ( ! isset( $title ) && empty( $track ) && ! isset( $track['title'] ) ) {
 	return;
+} elseif ( isset( $track['link'] ) && ! empty( $track['link'] ) ) {
```

Comment:
I would probably avoid `elseif`. Just have the single `if` at the top to exit early if the required vars are not available.

Then below just have a ternary to set the `$track_link` var like you had within the HTML attr in the original version.

Reply:
> Then below just have a ternary to set the $track_link var like you had within the HTML attr in the original version.

OK, just did.